angular-workspace/projects/ni/nimble-angular/src/thirdparty/directives/control_value_accessor.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * [Nimble]
+ * Copied from https://github.com/angular/angular/blob/14.2.6/packages/forms/src/directives/control_value_accessor.ts
+ */
+
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Directive, ElementRef, InjectionToken, Renderer2} from '@angular/core';
+
+/**
+ * @description
+ * Defines an interface that acts as a bridge between the Angular forms API and a
+ * native element in the DOM.
+ *
+ * Implement this interface to create a custom form control directive
+ * that integrates with Angular forms.
+ *
+ * @see DefaultValueAccessor
+ *
+ * @publicApi
```

Comment:
Similar the the other discussion, this is a public interface from Angular itself, we should not be redefining it and should extend from the Angular one instead. I think it would be reasonable to block comment out this interface

Reply:
Commented out. Because the code I was block commenting already contained a bunch of block comments, I had to un-terminate those comments.